support-frontend/assets/components/personalDetails/personalDetails.tsx

```diff
@@ -132,5 +132,3 @@ export function PersonalDetails({
 		</div>
 	);
 }
-
-export default PersonalDetails;
```

Comment:
Weird that this was available as both a default and a named export!

Reply:
Yeah an odd one